src/appdistribution/client.ts

```diff
@@ -42,10 +54,12 @@ export class AppDistributionClient {
 
   constructor(private readonly appId: string) {}
 
-  async getApp(): Promise<AppDistributionApp> {
-    utils.logBullet("getting app details...");
-
-    const apiResponse = await api.request("GET", `/v1alpha/apps/${this.appId}`, {
+  async getApp(
+    distributionFileType: DistributionFileType = DistributionFileType.APK
+  ): Promise<AppDistributionApp> {
+    utils.logBullet(`Getting app details (Distribution type: ${distributionFileType})...`);
```

Comment:
typically log lines are lowercased

```suggestion
    utils.logBullet(`getting app details (Distribution type: ${distributionFileType})...`);
```

Reply:
Fixed!